getter/getter.go

```diff
@@ -73,6 +73,10 @@ func AsyncGetBlocks(ctx context.Context, sessctx context.Context, keys []cid.Cid
 		return out, nil
 	}
 
+	// TODO: It's possible for a Session to miss a block if it requests the block
+	// after Bitswap already fetched it. May need to have an LRU of recently fetched blocks
+	// to cover this case
+
```

Comment:
Let's look into this.

Reply:
I've changed the code since I wrote this comment and as far as I can tell it's no longer an issue